### PR TITLE
QUnit support, fixes #3 :tada:

### LINKED ISSUE #3: Decouple from Mocha, support QUnit

### PR DESCRIPTION
**Pull request by lolmaus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +38,7 @@
     "ember-cli-htmlbars": "^3.0.0",
     "ember-cli-htmlbars-inline-precompile": "^1.0.3",
     "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-mirage": "^1.1.4",
```
Comment (simonihmig) [inline package.json:42]: Seems wrong having it under `dependencies`, shouldn't be `devDependencies`?

File: package.json [hunk 2]
```diff
@@ -52,7 +55,8 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.8.0"
+    "qunit-dom": "^0.8.0",
+    "sinon": "^7.5.0"
```
Comment (simonihmig) [inline package.json]: Where is it used?

File: package.json [hunk 3]
```diff
@@ -52,7 +55,8 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.8.0"
+    "qunit-dom": "^0.8.0",
```
Comment (simonihmig) [inline package.json]: Do we use `qunit-dom` actually?

**Comment by simonihmig** [comment]
Noticed CI is failing?
```
jsdom@12.2.0: The engine "node" is incompatible with this module. Expected version ">=8". Got "6.17.1"
```

Idk what dependency brings `jsdom` (as it seems unused). But I think we could also bump our minimum node version to 8 safely!

File: package.json (hunk 3 above)
Comment (lolmaus) [reply package.json]: Yep, for asserting attributes and classes, maybe something else.

File: package.json (hunk 2 above)
Comment (lolmaus) [reply package.json]: It's not, I have not included unit tests as they require much more work. Still not fully figured out how to approach them.

Comment (lolmaus) [reply package.json]: Removed.

**Comment by lolmaus** [comment]
> Noticed CI is failing?

Resolved by upgrading Ember CLI and dependencies.

**Review by simonihmig — APPROVED** [review]
Nice work!

**Pull request merged**